quickshow/src/cellrenderer.cpp

```diff
@@ -180,8 +183,8 @@ ResultsCellRenderer::ResultsCellRenderer(const QSize &size, QWidget *widget)
 	m_cellAttributes.resize(ColumnCount);
 	m_cellAttributes[Position] = CellAttribute{4 * m_scaledLetterWidth, Qt::AlignRight};
 	m_cellAttributes[Name] = CellAttribute{0};
-	m_cellAttributes[Registration] = CellAttribute{7 * m_scaledLetterWidth};
-	m_cellAttributes[Time] = CellAttribute{7 * m_scaledLetterWidth, Qt::AlignRight};
+	m_cellAttributes[Registration] = CellAttribute{5 * m_scaledLetterWidth};
```

Comment:
Why 7 is too much?

Reply:
On 4 cols and 100% of font size there is too much space, but before someone will find any possibility how to do it in general, we can leave it on 7.